SWLOR.Game.Server/Service/Fishing.cs

```diff
@@ -284,6 +284,7 @@ void CheckPosition(uint player, Vector3 startPosition, string attemptId)
             if (!GetIsObjectValid(fishingPoint) || GetIsDead(fishingPoint))
             {
                 SendMessageToPC(player, "This fishing point has been exhausted.");
+                Activity.ClearBusy(player);
```

Comment:
I'm skeptical of this because you're changing the state of the PC when, in reality, they shouldn't ever get to the Busy state in the first place.

I suspect the problem isn't with clearing the status but rather the GetIsDead check we're doing on line 284. I'll bet that only applies to creatures. Maybe it should be swapped from GetIsDead to GetCurrentHitPoints() < 0 instead. I'd recommend trying that before we start mucking with player state.